### PR TITLE
3266 Added a new round introductory call in codetrek rounds and made relat…

### PR DESCRIPTION
**Pull request by Priyanka-Kaintura**
Targets #3266

## Description of the changes

1. Added a new round introductory call to the codetrek rounds.
2. Made other related changes corresponding to introduction of new round.
3. Changed the default round of new codetrek applicant to 'introductory call' 

## Breakdown & Estimates 

**Expected Time Range:** 5 hrs

**Actual Time:** 4 hrs

## Feedback Cycle

**Cycle Count: 2**

## Checklist:

- [x] The PR title follows this syntax: <#IssueID> \<PR Title>
- [x] I have linked the issue id in the PR description.
- [x] I have performed a self-review of my own code.
- [x] I have added comments on my code changes where required.

### CODE REVIEW

File: Modules/CodeTrek/Services/CodeTrekService.php [hunk 1]
```diff
@@ -117,11 +117,11 @@ public function evaluate(CodeTrekApplicant $codeTrekApplicant)
         return $roundDetails;
     }
 
-    public function moveApplicantToRoundOne($applicant, $data)
+    public function moveApplicantToIntroductoryRound($applicant, $data)
     {
         $applicationRound = new CodeTrekApplicantRoundDetail();
         $applicationRound->applicant_id = $applicant->id;
-        $applicationRound->latest_round_name = 'level-1';
+        $applicationRound->latest_round_name = 'introductory-call';
```
Comment (LavishRawat) [inline Modules/CodeTrek/Services/CodeTrekService.php:124]: @Priyanka-Kaintura let’s pick this from config.

File: Modules/CodeTrek/Database/factories/CodeTrekApplicantRoundDetailsFactory.php [hunk 2]
```diff
@@ -28,7 +28,7 @@ public function definition()
             'applicant_id' => function () {
                 return CodeTrekApplicant::factory()->create()->id;
             },
-            'latest_round_name' => 'level-1',
+            'latest_round_name' => 'introductory-call',
```
Comment (LavishRawat) [inline Modules/CodeTrek/Database/factories/CodeTrekApplicantRoundDetailsFactory.php:31]: @Priyanka-Kaintura lets pick this from config.

**Review by LavishRawat — CHANGES_REQUESTED** [review]

**Comment by GauravGusain98** [comment]
@Priyanka-Kaintura This checklist is not yet completed
image

This checklist means that in your GitHub code changes you need to leave comments on line explaining the changes you have done

**Comment by Priyanka-Kaintura** [comment]
@GauravGusain98 Sir, I didn't find any piece of code requiring specific comment. So I haven't added any

File: Modules/CodeTrek/Config/config.php [hunk 3]
```diff
@@ -17,6 +17,11 @@
         ],
     ],
     'rounds' =>[
+        'introductory-call' => [
+            'label' => 'Introductory Call',
+            'slug'  => 'introductory-call',
+            'class' => 'badge badge-info'
```
Comment (Ayush8923) [inline Modules/CodeTrek/Config/config.php:23]: This color is already used in the `preparatory` round. Please use different color here.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Added one comment. Please check.

File: Modules/CodeTrek/Config/config.php [hunk 4]
```diff
@@ -17,6 +17,11 @@
         ],
     ],
     'rounds' =>[
+        'introductory-call' => [
+            'label' => 'Introductory Call',
+            'slug'  => 'introductory-call',
+            'class' => 'badge badge-warning'
```
Comment (Ayush8923) [inline Modules/CodeTrek/Config/config.php:23]: I think the `badge-warning` color does not make sense in the `introductory-call` round. I suggest make the custom class. and then used directly here.
Something like this.
```
.badge-theme-teal {
background-color: #63b2b7;
}
```
```suggestion
            'class' => 'badge badge-theme-teal'
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Added one more suggestion.

**Comment by Priyanka-Kaintura** [comment]
@Ayush8923 Please check the badge color now, if it is okay!

**Review by Ayush8923 — APPROVED** [review]
LGTM 👍🏻

**Review by LavishRawat — APPROVED** [review]

**Comment by GauravGusain98** [comment]
### Testing feedbacks
1. Instead of showing round slug here can you show label
image

**Pull request merged**